server/apps/main/forms.py

```diff
@@ -78,3 +84,89 @@ def clean_moderation_status(self):
             mod_status = 'not reviewed'
 
         return mod_status
+
+
+class ModerateExperienceForm(forms.Form):
+        
+    experience_text = forms.CharField(label='Please share your experience', max_length=500, strip=True,
```

Comment:
Similarly to including the `moderation_comments` above, I guess we don't actually need the `experience_text`, `difference_text` and `title_text` in this form, if we don't plan on allowing moderators to edit those.

Reply:
No but I think it is useful to have them in "form-form" at the moment?

Reply:
Yep, I think for now that's okay. It was mostly for what we want to work on going forward. Happy to have that merged in as is for now :)